scrapers/errata.py

```diff
@@ -69,6 +69,53 @@ def update_neo4j(self, advisories):
                 'update_date': advisory['update_date'],
                 'updated_at': advisory['updated_at']
             })[0]
+            container_adv = False
+
+            for associated_build in self.get_associated_builds(advisory['id']):
+                # Even if a node has two labels in the database, Neo4j returns the node
+                # only with the specific label you asked for. Hence we check for labels
+                # ContainerKojiBuild and KojiBuild separately for the same node.
+                build = ContainerKojiBuild.nodes.get_or_none(id_=associated_build['id_'])
+                if not build:
+                    build = KojiBuild.nodes.get_or_none(id_=associated_build['id_'])
+
+                if build and not container_adv:
+                    if build.__label__ == 'ContainerKojiBuild':
+                        adv.add_label(ContainerBuildAdvisory.__label__)
+                        container_adv = True
+
+                # If this is set, that means it was once part of the advisory but not anymore.
+                # This relationship needs to be deleted if it exists.
+                if associated_build['removed_index_id']:
+                    if build:
+                        adv.attached_builds.disconnect(build)
+                else:
+                    # This key shouldn't be stored in Neo4j
+                    del associated_build['removed_index_id']
```

Comment:
This can be removed since you don't use the variable directly anymore. You only select the `id` key to create a stub in Neo4j.